html/controls/mobile.html

```diff
@@ -46,16 +47,16 @@ <h1>Jam Control</h1>
 		<div class="ui-grid-a Period">
 			<div class="ui-block-a PeriodNumber">
 				<div class="Title">Period</div>
-				<div class="Number"><a class="Number"></a></div>
+				<div class="Number"><a class="Number" sbDisplay="Clock(Period).Number"></a></div>
 			</div>
-			<div class="ui-block-b PeriodTime"><a class="Time"></a></div>
+			<div class="ui-block-b PeriodTime"><a class="Time" sbDisplay="Clock(Period).Time" sbModify="toTime"></a></div>
 		</div>
-		<div class="JamTime Time"><h2>Jam Time</h2><a class="Time"></a></div>
-		<div class="LineupTime Time"><h2>Lineup Time</h2><a class="Time"></a></div>
-		<div class="TimeoutTime Time"><h2>TIMEOUT Time</h2><a class="Time"></a></div>
+		<div class="JamTime Time"><h2>Jam Time</h2><a class="Time" sbDisplay="Clock(Jam).Time" sbModify="toTime"></a></div>
+		<div class="LineupTime Time"><h2>Lineup Time</h2><a class="Time" sbDisplay="Clock(Lineup).Time" sbModify="toTime"></a></div>
+		<div class="TimeoutTime Time"><h2>TIMEOUT Time</h2><a class="Time" sbDisplay="Clock(Timeout).Time" sbModify="toTime"></a></div>
 		<div class="ui-grid-a">
 			<div class="ui-block-a">
-				<button class="StartJam" data-role="button" data-icon="check" data-iconpos="right"><span class="StartLabel Label">Start Jam</span></button>
+				<button class="StartJam" data-role="button" data-icon="check" data-iconpos="right"><span class="StartLabel Label")>Start Jam</span></button>
```

Comment:
What's the purpose of this unmatched closing paren?

Reply:
Must have been left over from me trying a few things, removed.